### DISCUSSION TITLE
صفحه پرداخت

### DISCUSSION THREAD

**Discussion by ft8680** [woocommerce/woocommerce#48751] [Checkout Flow]
سلام بخید میشه صفحه پرداخ من رو حرفه ای طراحی کنید و بعضی از بخش ها رو حذف کنید مثل گذینه حمل ونقل

**Comment by nielslange** [comment]
Hello @ft8680 👋

I've translated your message into English:

> Hello, can you professionally design my payment page and remove some parts, such as the transport option?

Could you please provide more details about what exactly you want to change? If you want to do it yourself, sharing more specific information will help us guide you better. Are you looking for someone to help you, or would you like to know how to make these changes on your own?

Thank you!
